pkg/bff/server.go

```diff
@@ -281,6 +281,7 @@ func (s *Server) setupRoutes() (err error) {
 		v1.GET("/lookup", s.Lookup)
 		v1.POST("/register/:network", s.Register)
 		v1.GET("/verify", s.VerifyContact)
+		v1.GET("/overview", auth.Authorize("read:vasp"), s.Overview)
```

Comment:
@masskoder let me know if you need any assistance with authentication and authorization.